spec/swagger_schemas/models/application.rb

```diff
@@ -0,0 +1,445 @@
+APPLICATION = {
+  v1: {
```

Comment:
could we revisit nullable vs required fields to match the current schemas, example `course_identifier` will never be null etc.